core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeExportService.java

```diff
@@ -179,7 +184,8 @@ protected EnrolleeExportData loadEnrolleeData(Enrollee enrollee) {
                 participantTaskService.findByEnrolleeId(enrollee.getId()),
                 surveyResponseService.findByEnrolleeId(enrollee.getId()),
                 kitRequestService.findByEnrollee(enrollee),
-                enrolleeRelationService.findByTargetEnrolleeIdWithEnrollees(enrollee.getId())
+                enrolleeRelationService.findByTargetEnrolleeIdWithEnrolleesAndFamily(enrollee.getId()),
```

Comment:
Gating these on the StudyEnvironmentConfig.enableFamilyLinkage will save a couple thousand DB queries for most studies.

Reply:
Refactored proxies to also only be grabbed when proxy enrollment is enabled + added tests for these.